### PR TITLE
Support Symfony 7

### PR DESCRIPTION
**Pull request by trsteel88**
| Q             | A
| ------------- | ---
| Branch?       | "master" for new features / the branch of the current release for fixes
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by dbu** [comment]
thank you for getting this started.

phpcr-bundle is ongoing in https://github.com/doctrine/DoctrinePHPCRBundle/pull/387 (and depends on more things...)

**Comment by trsteel88** [comment]
Hey @dbu, it looks like https://github.com/doctrine/DoctrinePHPCRBundle/pull/387 was merged. Do you think this one could be finalised soon?

File: composer.json [hunk 1]
```diff
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
```
Comment (dbu) [inline composer.json:17]:
```suggestion
        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || ^3.0@beta",
```
the new things are only in the 3.x branch of the bundle. i only did a beta release of it so far. lets install that to see if it works.

File: composer.json [hunk 2]
```diff
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
+        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || ^3.0@beta",
```
Comment (dbu) [inline composer.json:17]:
```suggestion
        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || 3.0.0-beta2",
```

File: composer.json [hunk 3]
```diff
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
+        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || 3.0.0-beta2",
         "doctrine/phpcr-bundle": "^1.3 || ^2.0.0",
```
Comment (dbu) [inline composer.json:18]: ah sorry, wrong bundle :facepalm: 

```suggestion
        "doctrine/phpcr-bundle": "^1.3 || ^2.0.0 || ^3.0@beta",
```

File: composer.json [hunk 4]
```diff
@@ -11,19 +11,19 @@
     "require": {
         "php": "^7.1|^8.0",
         "doctrine/data-fixtures": "^1.2",
-        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0"
+        "symfony/browser-kit": "^3.4.26 || ^4.3.8 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
+        "doctrine/doctrine-bundle": "^1.8 || ^2.0 || 3.0.0-beta2",
```
Comment (dbu) [inline composer.json:17]:
```suggestion
        "doctrine/doctrine-bundle": "^1.8 || ^2.0",
```

**Comment by dbu** [comment]
now we install with symfony 7 and see the issues.

@trsteel88 would you be motivated to look into the failing builds and try to solve the issues? i think its symfony 6 -> 7 BC breaks and in the symfony 6 build some incorrect mocking of the concrete class instead of the interface.

**Comment by dbu** [comment]
thanks for the help. i wrapped up and merged in #215

**Pull request closed without merging**